### PR TITLE
[NONE] Added a web-ui

### PR DESCRIPTION
**Pull request by maxnachlinger**
Adds a simple web-ui for transforming JSON -> GraphQL

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,11 +36,28 @@
     "pascal-case": "2.0.1"
   },
   "devDependencies": {
```
Comment (maxnachlinger) [inline package.json:38]: Libs used to build the web-ui

File: package.json [hunk 2]
```diff
@@ -23,7 +23,9 @@
     "format": "$(npm bin)/prettier --write './**/*.js'",
     "pretest": "npm run format && npm run lint",
     "lint": "$(npm bin)/eslint .",
-    "test": "$(npm bin)/jest --coverage."
+    "test": "$(npm bin)/jest --coverage.",
```
Comment (maxnachlinger) [inline package.json]: Fun new web-ui build targets

File: web-ui/src/index.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+      const schema = jsonToSchema({ jsonInput: jsonTextarea.value });
+      graphqlTextarea.value = schema.value;
+    } catch (error) {
+      jsonTextarea.classList.add("invalid");
+      graphqlTextarea.value = `Bad JSON:\n${error.message}`;
+      return;
+    }
+  };
+
```
Comment (maxnachlinger) [inline web-ui/src/index.js:28]: Don't parse the JSON -> GraphQL until the user has stopped typing for 500 ms

File: web-ui/webpack.prod.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
+/**
```
Comment (maxnachlinger) [inline web-ui/webpack.prod.js:1]: Builds `docs/index.html` which we'll hook gh-pages up to. The JS and CSS are inlined into the resulting index.html file as well - its all about 48k (most of the weight coming from `json-to-simple-graphql-schema` itself.

File: package.json (hunk 2 above)
Comment (ChrisCrewdson) [reply package.json]: Shouldn't that be `--coverage .` with a space? I'll have to try locally.

**Review by ChrisCrewdson — APPROVED** [review]
Looks good to me. One minor comment, need to try it out.

**Pull request merged**